src/utils/power_flow/make_pf.jl

```diff
@@ -60,6 +66,7 @@ function make_pf(system)
         bus_voltage = get_voltage(b)::Float64
         P_GEN_BUS[ix] = 0.0
         Q_GEN_BUS[ix] = 0.0
+        get_ext(b)["neighbors"] = neighbors[ix]
         for gen in get_components(StaticInjection, system, d -> !isa(d, ElectricLoad))
```

Comment:
There is another optimization possible here. We are looping all the generators for all the buses.